fauna/client.py

```diff
@@ -232,9 +231,8 @@ def _query(
             if opts.traceparent is not None:
                 headers[Header.Traceparent] = opts.traceparent
             if opts.query_timeout is not None:
-                headers[
-                    Header.
-                    TimeoutMs] = f"{opts.query_timeout.total_seconds() * 1000}"
+                headers[Header.TimeoutMs] = \
+                    f"{opts.query_timeout.total_seconds() * 1000}"
```

Comment:
Let's avoid manual formatting and tune yapf when we can. This way whatever tuning we do will have an effect.